src/mpcCoreKit.ts

```diff
@@ -443,10 +443,10 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
     }
   }
 
-  public async createFactor(createFactorParams: CreateFactorParams): Promise<string> {
+  public async createFactor(params: CreateFactorParams): Promise<string> {
```

Comment:
If you're looking to replace all with just `params`, in that case please update `enableMFAParams` and all others that are using similar naming convention. Either way works for us, but it should be consistent, no matter what we use.

Reply:
see reply above